Ports/libopenal/patches/0001-Disable-example-programs-and-force-SDL2-backend-to-b.patch

```diff
@@ -0,0 +1,37 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: cflip <cflip@cflip.net>
+Date: Mon, 17 Oct 2022 19:21:52 -0600
+Subject: [PATCH] Disable example programs and force SDL2 backend to build
+
+---
+ CMakeLists.txt | 9 ++++++---
+ 1 file changed, 6 insertions(+), 3 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0cf0613d65d2d36425308128e566f700de812fd8..d8abc66f6c249acba42d1f2e9577fc5666206e6d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -77,9 +77,9 @@ option(ALSOFT_DLOPEN  "Check for the dlopen API for loading optional libs"  ON)
+ option(ALSOFT_WERROR  "Treat compile warnings as errors"      OFF)
+ 
+ option(ALSOFT_UTILS "Build utility programs"  ON)
+-option(ALSOFT_NO_CONFIG_UTIL "Disable building the alsoft-config utility" OFF)
++option(ALSOFT_NO_CONFIG_UTIL "Disable building the alsoft-config utility" ON)
+ 
+-option(ALSOFT_EXAMPLES  "Build example programs"  ON)
++option(ALSOFT_EXAMPLES  "Build example programs"  OFF)
```

Comment:
There's no reason whatsoever to patch out CMake options we can simply pass to the `run cmake` invocation :thinking: